build.bat

```diff
@@ -18,7 +17,9 @@ set vcvars64="C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\V
 set CLICOLOR_FORCE=1
 
 if not defined DevEnvDir (
```

Comment:
What is the DevEnvDir?

Reply:
It is set by vcvars64. I check here to not re-run it everytime (expanding the PATH to infinity - related to #5).